src/Raven.Client/ServerWide/Operations/Logs/SetLogsConfigurationOperation.cs

```diff
@@ -16,6 +16,7 @@ public class SetLogsConfigurationOperation : IServerOperation
         public class Parameters
         {
             public LogMode Mode { get; set; }
+            public TimeSpan LogRetention { get; set; }
```

Comment:
rename to `RetentionTime`

Reply:
Also needs to be nullable to maintain backward compatibility

Reply:
done